evaluate_tflite.py

```diff
@@ -79,27 +83,39 @@ def main():
 
     print([x.name for x in processes])
 
+    wavlist = []
     ground_truths = []
     predictions = []
     losses = []
 
     with open(args.csv, 'r') as csvfile:
         csvreader = csv.DictReader(csvfile)
+        count = 0
         for row in csvreader:
+            count += 1
             work_todo.put({'filename': row['wav_filename'], 'transcript': row['transcript']})
+    print('Totally %d work todo\n' % count)
     work_todo.join()
+    print('\nTotally %d work done' % work_done.qsize())
```

Comment:
Are the `count` variable and these prints also debugging leftover?

Reply:
One of the redundant 'Total' can be removed.

Reply:
I would keep just the latter I guess.